server/server.go

```diff
@@ -71,6 +73,12 @@ const defaultCapability = mysql.ClientLongPassword | mysql.ClientLongFlag |
 	mysql.ClientTransactions | mysql.ClientSecureConnection | mysql.ClientFoundRows |
 	mysql.ClientMultiStatements | mysql.ClientMultiResults | mysql.ClientLocalFiles |
 	mysql.ClientConnectAtts | mysql.ClientPluginAuth
+const (
+	// MysqlProtocol is MySQL Protocol
+	MysqlProtocol = 1
+	// MysqlXProtocol is MySQL X Protocol
+	MysqlXProtocol = 2
```

Comment:
Consider to make a type alias for this two constants.

Reply:
Done.